util/CMakeLists.txt

```diff
@@ -0,0 +1,44 @@
+set_source_path(UTIL_SRCS
+  al_info.cpp)
+
+set_source_path(UTIL_HDRS
+)
+
+if (AL_HAS_ROCM)
+  hipify_header_files(AL_UTIL_HEADERS ${UTIL_HDRS})
+  hipify_source_files(AL_UTIL_SOURCES ${UTIL_SRCS})
```

Comment:
The hipify/special ROCm hand-holding isn't strictly necessary here. But it could be needed if more complex utils are added at a later date, so I don't object to keeping it if there's a real possibility of that. It's not hurting anything (should be a no-op if there's no CUDA code in the original source).

Reply:
Makes sense. I also didn't think it was necessary, but I mainly put it in so we don't have to remember to add it if we do some ROCm-oriented utilities in the future.